server/db/migrate/20150429180751_add_root_passwords_to_deployments.rb

```diff
@@ -0,0 +1,6 @@
+class AddRootPasswordsToDeployments < ActiveRecord::Migration
+  def change
+    add_column :fusor_deployments, :rhev_root_password, :string
+    add_column :fusor_deployments, :cfme_root_password, :string
```

Comment:
This means that rhev engines and hypervisors will be the same password. Is this the intent as well?
For UX, not sure if there should be an option to overwrite per host and I save it in the `root_pass` on the `hosts` table?